src/interfaces/IFixedRateIrm.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.5.0;
+
+import {IIrm} from "../../lib/morpho-blue/src/interfaces/IIrm.sol";
+import {Id} from "../../lib/morpho-blue/src/interfaces/IMorpho.sol";
+
+/// @title IAdaptiveCurveIrm
+/// @author Morpho Labs
+/// @custom:contact security@morpho.org
+interface IFixedRateIrm is IIrm {
+    /// @notice Sets the borrow rate for a market.
+    /// @dev A rate can be set by anybody, but only once.
```

Comment:
maybe add a comment to say that market creation should be batched with the setting of the rate?